content/kotlin/concepts/arrays/terms/map/map.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '.map()'
+Description: 'map() is a Kotlin method that transforms elements in a collection.'
```

Comment:
```suggestion
Description: '`map()` is a Kotlin method that transforms elements in a collection.'
```
.map( ) is a function so it should be written inside the single backticks(`.....`)